.gitignore

```diff
@@ -0,0 +1,11 @@
+# IDEs
+.idea/
+.vscode/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# macOS crap
```

Comment:
Minor: remove crap here and add trailing new line

Reply:
Crap removed, newline added.